CMakeLists.txt

```
@@ -185,6 +185,7 @@ option(USE_TSAN "Use Thread Sanitizer" OFF)
 option(USE_CUDA "Use CUDA" ON)
 cmake_dependent_option(
      BUILD_LAZY_CUDA_LINALG "Build cuda linalg ops as separate library" ON "USE_CUDA AND LINUX AND BUILD_PYTHON" OFF)
+option(BUILD_NVFUSER "Build NVFUSER" ON)
```

Can it be `cmake_dependent_option`
```suggestion
cmake_dependent_option(BUILD_NVFUSER "Build NVFUSER"  ON "USE_UCDA OR USE_ROCM" OFF)
```